content/en/real_user_monitoring/guide/browser-sdk-upgrade.md

```diff
@@ -10,9 +10,193 @@ further_reading:
   text: "Use Datadog Session Replay to view real-time user journeys"
 ---
 
-## Overview 
+## Overview
 
-Follow this guide to migrate between major versions of the RUM and Logs Browser SDKs.
+Follow this guide to migrate between major versions of the Browser RUM and Browser Logs SDKs.
+
+## From v4 to v5
+
```

Comment:
Let's add the availability date here:

V5 will be available starting October XX, 2023.